SwiftWisdom/Core/Foundation/URL+Extensions.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  URL+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Max Litteral on 7/17/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+extension URL: ExpressibleByStringLiteral {
+
+    public init(stringLiteral value: StaticString) {
+        self = URL(string: "\(value)")!
```

Comment:
or, can add https://github.com/JohnSundell/Require pod, which looks like the app will still crash if a bad URL string is used but can attach a useful message.